cmake/helpers/GdalGenerateConfig.cmake

```diff
@@ -44,6 +44,11 @@ function(gdal_flatten_link_libraries _result)
         elseif(_lib STREQUAL "general")
             continue()
         endif()
+        if(_lib IN_LIST _seen_libs)
+            # Library has already been processed
+            continue()
+        endif()
+        list(APPEND _seen "${_lib}")
```

Comment:
shouldn't that be
```suggestion
        list(APPEND _seen_libs "${_lib}")
```

Reply:
Oops, looks like I messed up when renaming it for the PR. Sorry about that!